.github/workflows/consecutive.yml

```diff
@@ -100,7 +100,7 @@ jobs:
 
   openzeppelin:
     timeout-minutes: 360
-    runs-on: self-hosted
+    runs-on: ubuntu-20.04
```

Comment:
before merge, return this ti self-hosted

Reply:
of course, I'm just not able to run on the self-hosted runner from the fork